articles/api/authentication/api-authz/_highly-regulated.md

```diff
@@ -0,0 +1,213 @@
+# Highly Regulated Identity
+## Push Authorization Requests (PAR)
+
+<%= include('../../../_includes/_http-method', {
+  "http_badge": "badge-primary",
+  "http_method": "POST",
+  "path": "/oauth/par",
+  "link": "##push-authorization-requests-par-"
+}) %>
+
+```http
+POST ${account.namespace}/oauth/par
+Content-Type: 'application/x-www-form-urlencoded'
+  response_type=code|code id_token&
+  client_id={yourClientId}&
+  redirect_uri={https://yourApp/callback}&
+  state=STATE&
+  scope=openid|profile|email&
+  code_challenge=CODE_CHALLENGE&
+  code_challenge_method=S256&
+  nonce=NONCE&
+  connection=CONNECTION&
+  prompt=login|consent|none&
+  organisation=ORGANIZATION
+```
+
+```javascript
+var request = require("request");
+
+var options = { method: 'POST',
+  url: 'https://{yourDomain}/oauth/par,
+  headers: { 'content-type': 'application/x-www-form-urlencoded' },
+  form: { 
+    response_type: 'code|code id_token',
```

Comment:
It is important that all example include
* an audience param (optional)
* an authorization_details param (optional)

Reply:
Pending updates on Google doc